src/site/xdoc/index.xml

```diff
@@ -43,7 +43,7 @@
 
       <section name="Releases">
          <p>
-            See the <a href="downloads.html">downloads</a> page for information
+            See the <a href="https://commons.apache.org/proper/commons-pool/download_pool.cgi">downloads</a> page for information
```

Comment:
Refer to content of `site.xml`
`<item name="Download" href="https://commons.apache.org/proper/commons-pool/download_pool.cgi" />`